specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2019-12-12/cosmos-db.json

```diff
@@ -6530,6 +6530,12 @@
     },
     "CreateUpdateOptions": {
       "type": "object",
+      "properties": {
+        "throughput": {
+          "type": "string",
```

Comment:
Value as a string?

Reply:
Yes, the all the value are string before this change. We don't change the value type, just want to specify the property name "throughput".